### DISCUSSION TITLE
Negative Lookbehind Support in Access Rules

### DISCUSSION THREAD

**Discussion by wsimonTT** [ory/oathkeeper#955] [Q&A] [answered]
Is there a way to use negative lookbehind with a regex access rule matching pattern? I see that it's supported by the `dlclark/regexp2` package that Oathkeeper uses, but whenever I try to use it within a pattern, I get an error:

Pattern: `https://localhost:4455/api/portal/v<[0-9]+>/<.*(?<!users\/authenticate)$>`
Error Message:  `Unbalanced braces in \"\"https://localhost:4455/api/v<[0-9]+>/<.*(?<!users/authenticate)$>\"\"`

If I try to escape the internal `<` character in the regex pattern, a different error is thrown. 

Is this intended to be a supported feature?

**Comment by aeneasr** [comment, marked as answer]
It's not possible to use start/end control characters `$` and `^` as those are applied automatically by `<>` :)

**Reply by vinckr** [reply]
Hey @wsimonTT,
Just checking in if your question was answered or if there was still something unclear.
I will mark this as answered for now, but feel free to come back to the discussion any time.

Thanks a ton! 🐝
